src/lib.ts

```diff
@@ -61,6 +62,7 @@ export const FetchFlow = FetchFlowComponent
 export const NewFlow = NewFlowComponent
 export const ImportFlow = ImportFlowComponent
 export const Home = HomeComponent
+export const TreeBuilderToolbar = TreeBuilderToolbarComponent
```

Comment:
Honestly, a lot of this comes through trial and error rather than following a specific pattern. Very happy to be corrected if there is a better approach which achieves the same goal. Here I think I just wanted the `XComponent` `const`s so I could use them twice. 

One this that might be good to add here is an index file of everything in the toolbar directory in the same way as commonComponents. Long term we want almost everything overridable - See VMO-442